test/index.js

```diff
@@ -107,7 +107,8 @@ describe('Basic behaviors', function() {
     done();
   });
 
-  it('treats 1st argument as an empty object when it is invalid', function(done) {
+  it('treats 1st argument as an empty object when it is invalid',
```

Comment:
eslint raises warnings of "max-len". So I added an eslint comment to `test/index.js` for ignoring line length, instead.